superset-frontend/plugins/plugin-chart-echarts/src/Gauge/EchartsGauge.tsx

```diff
@@ -77,7 +77,7 @@ export default function EchartsGauge({
         const pointerEvent = e.event.event;
         const filters: QueryObjectFilterClause[] = [];
         if (groupby.length > 0) {
-          const values = e.name.split(',');
```

Comment:
@michael-s-molina This was breaking for single values that had commas in them - does my replacement make sense?

Reply:
If you mean that we have single values with a dangling comma at the end, then your code will fix it. However, if we have values with commas then we need to use a different separator. Let me know which one is it and if it's the separator case then I can open a PR to fix it for all charts.

Reply:
@michael-s-molina It was for values with commas inside – the one it was breaking on was in the FCC New Coder Survey 2018 dashboard, Minority Status (in their city) chart, `Yes, an ethnic minority` value.  If you could fix this in a separate PR that would be great, thanks!

Reply:
Ok. I'll create ticket/PR for it.

Reply:
Thanks, I'll drop the commit that added these fixes from this PR.